lib/Backend/Lower.cpp

```diff
@@ -9783,6 +9783,14 @@ Lowerer::LowerEqualityBranch(IR::Instr* instr, IR::JnHelperMethod helper)
     }
     if (!needHelper)
     {
+        if (instr->AsBranchInstr()->GetTarget()->m_isLoopTop)
+        {
+            instr->AsBranchInstr()->GetTarget()->m_isLoopTop = false;
```

Comment:
Is this necessarily the right change? In the case of continue statements, we may have multiple back-edges, and I think we'd still need to have the isLoopTop annotation in that scenario.